### ISSUE TITLE
Get entities at a location

### ISSUE THREAD

**Issue by latibro** [latibro/Automation-Mod#5]
Able to get a EntityList of entities at a location

**Comment by latibro** [comment]
Maybe something like this

```lua
local entities = location.getEntities()
```

**Comment by latibro** [comment]
This is related to #4  
But this issues talks about entities at the exact location, where #4 talks about entities within a range of the location

**Comment by latibro** [comment]
Included in 0.1.0.alpha.4

```lua
location.getEntities() : EntityCollection
```
